app/assets/javascripts/angular/controllers/dashboard.js

```diff
@@ -0,0 +1,172 @@
+angular.module('dashboardApp', ['timer', 'ngAnimate'])
```

Comment:
I think your rebase adding something it should not have added...?  Can you please double-check?